tests/LSP6KeyManager/tests/PermissionSetData.test.ts

```diff
@@ -201,6 +201,29 @@ export const shouldBehaveLikePermissionSetData = (
             .withArgs(cannotSetData.address, "SETDATA");
         });
       });
+
+      describe("when sending value while setting data", async () => {
+        it("should revert", async () => {
```

Comment:
let's add the error name in the test name to be specific.

```suggestion
        it("should revert with Key Manager error `CannotSendValueToSetData`", async () => {
```